VideoWeb/VideoWeb.AcceptanceTests/Pages/HearingRoomPage.cs

```diff
@@ -5,7 +5,7 @@ namespace VideoWeb.AcceptanceTests.Pages
     public static class HearingRoomPage
     {
         public static readonly By PauseButton = By.Id("pause-hearing");
-        public static readonly By CloseButton = By.Id("end-hearing");
+        public static readonly By CloseButton = By.Id("end-hearing-landscape");
```

Comment:
This seems to imply that there will be a new button to be added called end-hearing-portrait, maybe you need to rename it to CloseButtonLandscape as a CloseButtonPortrait is coming?